xprof/interval_model.yaml

```
@@ -53,4 +53,33 @@
   - :name: name
     :class: string
   - :name: size
-    :class: unsigned 
+    :class: unsigned
+- :name: lttng:frequency
+  :payload:
+  - :name: did
+    :class: unsigned
+    :class_properties:
+      :preferred_display_base: 16
+  - :name: domain
+    :class: unsigned
+    :class_properties:
+      :preferred_display_base: 16
+  - :name: frequency
+    :class: unsigned
+    :class_properties:
+      :preferred_display_base: 16
```

```suggestion
  - :name: frequency
    :class: unsigned
```
Frquency (and power) don't really benefit from being displayed in hexadecimal notation.